### ISSUE TITLE
[v3-alpha] the demo doesn't appear to run

### ISSUE THREAD

**Issue by esistgut** [marmelab/react-admin#3436]
Looks like it is using the v3 features while keeping v2 dependencies in `package.json`.
Forcing the new versions results in the following error:
`Unhandled Rejection (Error): Invariant failed: You should not use <Switch> outside a <Router>` on `src/App.js:45`
```
  42 |         process.env.REACT_APP_DATA_PROVIDER
  43 |     );
  44 | 
> 45 |     this.setState({ dataProvider });
     | ^  46 | }
  47 | 
  48 | componentWillUnmount() {
```

**Comment by Kmaschta** [comment]
Locally, you have to rebuild everytime you change the react-admin code before running the demo.

Can you check it by running `make install build run-demo` on your host?

**Comment by fzaninotto** [comment]
The bug is real - I release 3.0.0-alpha.0 a bit too fast. #3440 should solve it.

**Comment by esistgut** [comment]
The demo is now running, thank you.

**Comment by fzaninotto** [comment]
Fixed by #3440
